docs/source/img/rem_workflow.svg

```diff
@@ -0,0 +1,383 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
I'm not sure what happened, but it looks like the arrows are missing from this diagram.

Reply:
I'm seeing arrows here:
<img width="1194" alt="Screen Shot 2022-12-26 at 12 40 09 PM" src="https://user-images.githubusercontent.com/904110/209582139-f7c3f6f4-6b2b-4eab-9d0b-f227f347d40f.png">

You're not?

Reply:
Strangely, no!

<img width="613" alt="Screenshot 2022-12-26 at 11 55 33 PM" src="https://user-images.githubusercontent.com/12703123/209632245-ba6333a1-c4a5-42c8-af9c-ca038ab4b9c7.png">

After testing on Chromium it seems to work there, but not on firefox. What browser are you on?

Reply:
I'm on Brave, which is Chromium. Do you have the latest Firefox version?

Reply:
I tried with Firefox 72.0.2 on OSX and it rendered correctly:
<img width="1406" alt="Screen Shot 2022-12-27 at 7 50 03 PM" src="https://user-images.githubusercontent.com/904110/209754472-11e50883-1c37-493b-8d90-dbc92db18956.png">

Reply:
Firefox 108.0.1 (64-bit) on Ubuntu 22.04 LTS doesn't render arrows, either, rem_workflow.svg

Reply:
Thanks for checking Dan. I'm also on Firefox 108.0 (64-bit) on macOS Ventura 13.0.

Amir, if you upgrade firefox does it still work?

Reply:
With Firefox 108.0.2 it does on macOS Catalina 10.15.7 it doesn't render the arrows. I'll see if I can track down why.

Reply:
Should be fixed now. The trick is to use "Stroke to Path" feature in Inkscape, which splits the line and arrow out to separate shapes. I kept the original line/arrow in there, too, as a source and just hid the layer.

Reply:
<img width="1095" alt="image" src="https://user-images.githubusercontent.com/904110/212456705-1993f0b7-4b23-49a6-9225-c477dac3ed70.png">